src/interface.rs

```diff
@@ -224,6 +224,12 @@ impl Interface {
         Ok(interface)
     }
 
+    pub fn from_str(s: &str) -> Result<Self, Error> {
```

Comment:
Why redefine this function? It is already defined here https://github.com/astarte-platform/astarte-device-sdk-rust/blob/5d4ec0a5dc5736382e2f27d53fb1be1f2892f6f8/src/interface.rs#L317